.changeset/shaggy-files-juggle.md

```diff
@@ -0,0 +1,5 @@
+---
+"@kadena/react-ui": patch
+---
+
+feat: new checkbox component added to the component library
```

Comment:
Would recommend reading the CONTRIBUTING.md file in the root of kadena.js 

Here's a snippet about changesets: 
```#### Authoring Changelogs

To keep everything clear for ourselves and our end users, we have a "changelog
etiquette". Only include changes that affect the consumer of your
package/app/product. It's not a commit log. Write descriptions that are
understandable from the consumers' perspective:

- Start descriptions with action verbs like "Add", "Remove", "Deprecate", "Fix",
  "Improve", "Update".
- Avoid the term "bug", use "issue" instead.
- Add GitHub issue numbers when fixing those.
- When referring to public API changes, use backticks and parentheses for code
  like function names and classes (e.g. `createClient()`,
  `new TransactionBuilder()`, `hash`).
- Upgrades should be documented with old and new version numbers.
- Avoid trailing periods (unless in a description below the commit title).

Examples:

- Deprecate the `doSomething()` API function.
- Use `doSomethingBetter()` instead.
- Fix issue where `ExampleWidget` API did not handle dates correctly (fix #81)
- Improve the diagnostic logging when running in advanced mode.
- Upgrade from `react@15` to `react@16-beta` release of the flexible panels
  feature

(Credits: [rushjs.io/pages/best_practices/change_logs][8])

If you don't see a need for authoring changelogs for your package (e.g. a PoC or
packages without consumers), add it to [.changeset/config.json#ignore][9].
```

Reply:
Clear, I'll edit this message. Thanks!